modules/ROOT/pages/creating-connected-apps-dev.adoc

```diff
@@ -0,0 +1,195 @@
+= Creating Connected Apps
+
+When you create connected apps, keep in mind: 
+
+* Your organization can own up to 2000 connected apps at the root organization or business group level.
+* Each connected app can have up to 1000 scopes.
+* All scopes in the endpoint map directly to permissions available in Anypoint Platform, except for exceptions noted in <<endpoint-scopes>>. Scopes determine the level of permissions to provide to the app. 
```

Comment:
I misread this as "endpoint map" (map as a noun not a verb). Suggest an alternative word like "correspond".